02-express-tutorial/app.js

```diff
@@ -1 +1,166 @@
-console.log('Express Tutorial')
+console.log("Express Tutorial");
+
+const express = require("express");
+const { products } = require("./data");
+const { people } = require("./data");
+const peopleRouter = require("./routes/people");
+const app = express();
+
+app.use(express.static("./public"));
+
+console.log("Static files are being served from ./public");
+
+const logger = (req, res, next) => {
+    console.log(`${req.method} ${req.url} at ${new Date().toISOString()}`);
+    next();
+};
+app.use(logger);
+
+app.use("/api/v1/people", peopleRouter);
+
+app.use(express.static("./methods-public"));
+app.use(express.json());
+
+app.get("/api/v1/people", (req, res) => {
+    res.json(people);
+})
+
+app.post("/api/v1/people", (req, res) => {
+    try {
+        const { name } = req.body;
+
+        if (!name) {
+            return res.status(400).json({ success: false, message: "Please provide a name" });
+        }
+
+        const newPerson = { id: people.length + 1, name };
+        people.push(newPerson);
+        
+        res.status(201).json({ success: true, data: newPerson });
+    } catch (error) {        
+        console.error("Error in POST /api/v1/people:", error);
+        
+        res.status(500).json({ success: false, message: "An error occurred while adding the person." });
+    }
+});
```

Comment:
This is no longer required, right?  Now that you have re-factored your people controllers into a separate file.